scripts/rollup/build.js

```diff
@@ -347,6 +347,7 @@ function getPlugins(
     bundleType === RN_FB_DEV ||
     bundleType === RN_FB_PROD ||
     bundleType === RN_FB_PROFILING;
+  const isTestBuild = entry === 'react-dom/testing';
```

Comment:
Shouldn't this also be true for React Noop and React Test Renderer?

Reply:
I thought we'd delay it until we actually used __TEST__ in any of its code paths. Happy to add it right now tho.

Reply:
This isn't necessary anymore, since the renderers read the feature flag now